build.sbt

```diff
@@ -153,6 +163,7 @@ addCommandAlias(
   Seq(
     s"+${bridgeIntegration.id}/$publishLocalCmd",
     s"+${zincIntegration.id}/$publishLocalCmd",
+    s"${bspIntegration.id}/$publishLocalCmd",
```

Comment:
While I'm thinking about it, is there a released version of `bsp` somewhere? We'll need it before the next release :)